tensorflow/compiler/xla/service/gpu/horizontal_loop_fusion.cc

```diff
@@ -493,16 +524,26 @@ StatusOr<bool> HorizontalLoopFusionImpl::Run() {
     auto consumer = def_to_use_order[i];
     HorizontalLoopFusionImpl::FusionCandidates fusion_candidates(consumer);
     while (true) {
```

Comment:
If we are running this in a fixed point, could we remove this while/true loop then?